tests/sniffer.py

```diff
@@ -73,6 +74,23 @@ def _simplify(x):
             return x
         return list(map(_simplify, packets))
 
+    def get_packets_for_cmdu_type(self, test_name, msg_type: int):
```

Comment:
I think we're missing an important argument here - message identifier (mid) - this one should be used to verify that we are looking at the correct cmdu. This should be optional parameter, so if its none, it should return all the CMDUs which match the message type, otherwise it would return a single CMDU or None if there is no message with this MID.

In addition, we must also add source and destination macs parameters, since this is the only way we can uniquely identify a message - source + destination ALid + mid.

Reply:
Yeah, there are multiple ways to go about it. Basically we want to check "was there a packet that matches these and these parameters".

I think actually the best way to do it would be to have a check_packet function in TestFlows itself that takes the filter as an argument. The filter can then check all the data it needs to.